locales/en/moves.json

```diff
@@ -185,5 +186,12 @@
   },
   "agreement_status": {
     "heading": "Has this move been agreed with receiving prison?"
+  },
+  "update_links": {
+    "court": "Change <span class=\"app-!-visually-hidden-desktop\">information for the court</span>",
```

Comment:
These all seem very similar? What about one key for the link, and passing the translations in separately so we don't have to duplicate the `Change <span>` each time?

Reply:
You could even use the keys that already exist for the headings of the sections that are being updated for consistency (maybe lowercasing using a nunjucks filter though)

Reply:
Done and reimplemented as a macro (declared in the view template). It seems much cleaner now.